hs_core/views/utils.py

```diff
@@ -536,8 +536,8 @@ def create_form(formclass, request):
     return params
 
 
-def get_my_resources_list(request):
-    user = request.user
+def get_my_resources_list(user):
```

Comment:
I'd approve this review just for this change.  It drives me nuts seeing request passed around everywhere.